vms/platformvm/vm.go

```diff
@@ -65,8 +64,7 @@ type VM struct {
 	*network.Network
 	validators.State
 
-	metrics            metrics.Metrics
-	atomicUtxosManager avax.AtomicUTXOManager
```

Comment:
only `service` uses `atomicUtxosManager`, so `atomicUtxosManager` should be a service member